pkg/dbutil/common.go

```diff
@@ -118,6 +119,10 @@ func OpenDB(cfg DBConfig) (*sql.DB, error) {
 		dbDSN = fmt.Sprintf("%s:%s@tcp(%s:%d)/?charset=utf8mb4", cfg.User, cfg.Password, cfg.Host, cfg.Port)
 	}
 
+	for key, val := range vars {
+		dbDSN += fmt.Sprintf("&%s=%%27%s%%27", key, url.QueryEscape(val))
```

Comment:
Can you add a comment on this to help others understand?

Reply:
Done